backend/src/schema/resolvers/embeds.spec.js

```diff
@@ -0,0 +1,216 @@
+              lang: 'de',
+              publisher: 'Facebook',
+              sources: ['resource'],
+              title: 'Human Connection',
+              type: 'link',
+              url: 'https://www.facebook.com/HumanConnectionOrg/',
+              video: null,
+            },
+          },
+        })
+        await expect(embedAction(variables)).resolves.toEqual(expected)
+      })
+    })
+
+    describe('given a Github link', () => {
+      beforeEach(() => {
+        fetch
+          .mockReturnValueOnce(Promise.resolve(new Response(pr960)))
+          .mockReturnValueOnce(Promise.resolve(JSON.stringify({})))
+        variables = { url: 'https://github.com/Human-Connection/Human-Connection/pull/960' }
+      })
+
+      it('returns meta data even if no embed html can be retrieved', async () => {
+        const expected = expect.objectContaining({
+          data: {
+            embed: {
+              type: 'link',
+              title:
+                'Editor embeds merge in nitro embed by mattwr18 · Pull Request #960 · Human-Connection/Human-Connection',
+              author: 'Human-Connection',
+              publisher: 'GitHub',
+              date: expect.any(String),
+              description: '🍰 Pullrequest Issues fixes #256',
+              url: 'https://github.com/Human-Connection/Human-Connection/pull/960',
+              image:
+                'https://repository-images.githubusercontent.com/112590397/52c9a000-7e11-11e9-899d-aaa55f3a3d72',
+              audio: null,
+              video: null,
+              lang: 'en',
+              sources: ['resource'],
+              html: null,
+            },
+          },
+        })
+        await expect(embedAction(variables)).resolves.toEqual(expected)
+      })
+    })
+
+    describe('given a youtube link', () => {
+      beforeEach(() => {
+        fetch
+          .mockReturnValueOnce(Promise.resolve(new Response(babyLovesCat)))
+          .mockReturnValueOnce(Promise.resolve(babyLovesCatEmbedResponse))
+        variables = { url: 'https://www.youtube.com/watch?v=qkdXAtO40Fo&t=18s' }
+      })
+
+      it('returns meta data plus youtube iframe html', async () => {
+        const expected = expect.objectContaining({
+          data: {
+            embed: {
+              type: 'video',
+              title: 'Baby Loves Cat',
+              author: 'Merkley Family',
+              publisher: 'YouTube',
+              date: expect.any(String),
+              description:
+                'She’s incapable of controlling her limbs when her kitty is around. The obsession grows every day. Ps. That’s a sleep sack she’s in. Not a starfish outfit. Al...',
+              url: 'https://www.youtube.com/watch?v=qkdXAtO40Fo',
+              image: 'https://i.ytimg.com/vi/qkdXAtO40Fo/maxresdefault.jpg',
+              audio: null,
+              video: null,
```

Comment:
`audio` and `video` was always `null` in all my tests...

Reply:
strange, that was what I observed as well

Reply:
Is should be filmed when there is video or audio contend in an article as far as I understand metascraper.

Reply:
*filled